src/Inkluzitron/Data/Entities/QuizItemBase.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Inkluzitron.Data.Entities
+{
+    public abstract class QuizItemBase<Result> where Result : QuizResultBase
+    {
+        [Key]
+        public long Id { get; set; }
+
+        [ForeignKey("ParentId")]
```

Comment:
```suggestion
        [ForeignKey(nameof(ParentId))]
```

Vyhnul bych se používání stringů. Když to pak někdy změníš, tak tě to donutí to změnit i jinde.